Solutions/Azure SQL Database solution for sentinel/Analytic Rules/Detection-VolumeResponseRowsStatefulAnomalyOnDatabase.yaml

```diff
@@ -42,9 +43,7 @@ query: |
                         , additional_information_s has 'failure_reason', toint(extract("<failure_reason>Err ([0-9.]+), Level ([0-9.]+)", 2, additional_information_s))
                         , 0)
         | extend QuantityColumn = column_ifexists(monitoredColumn, 0)
-        | sort by TimeGenerated desc
-        | extend RowNumber = row_number(),
```

Comment:
Removing this breaks the query as RowNumber is used in the Summarize after this.